src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,8 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
```

Comment:
This does pass the test, but you have changed the requirements of the test to assert for the **strings** `"true"` and `"false"` rather than the **booleans** `true` and `false`. The strings `"true"` and `"false"` are both **truthy** values. This has added complexity to this function by needing to conditionally checking for the string `"true"`.

If you had kept the test as it was, sticking with booleans (`true` or `false`), you could have simplified this function with the ternary operator:

```js
const booleanToWord = boolean => boolean ? "Yes" : "No"
```